inc/helpers/namespace.php

```diff
@@ -477,7 +477,8 @@ function get_links( $echo = true ) {
 
 	if ( $echo ) :
 		?>
-		<nav class="nav-reading <?php echo $multipage ? 'nav-reading--multipage' : '' ?>" role="navigation">
+		<nav aria-labelledby="reading-nav" class="nav-reading <?php echo $multipage ? 'nav-reading--multipage' : '' ?>" role="navigation">
+		<h2 id="reading-nav" class="screen-reader-text">Previous/next navigation</h2>
```

Comment:
Is this an ok heading title for what this element does? Open to better suggestions ...

Reply:
Yeah I'm not sure about the h2 inside the nav, because the h2 inside of the nav are not part of the "content" I feel because that is only used for screen reader purposes it could be better an span, what do you think?

Reply:
Agree. Changing to `<p>` as recommended by Ned.